src/editors/sharedComponents/TinyMceWidget/hooks.js

```diff
@@ -436,6 +440,15 @@ export const setAssetToStaticUrl = ({ editorValue, lmsEndpointUrl }) => {
     const updatedContent = content.replace(currentSrc, portableUrl);
     content = updatedContent;
   });
+
+  /* istanbul ignore next */
+  assetSrcs.filter(src => src.startsWith('static/')).forEach(src => {
+    const nameFromEditorSrc = parseAssetName(src);
+    const portableUrl = `/${ nameFromEditorSrc}`;
+    const currentSrc = src.substring(0, src.search(/("|&quot;)/));
```

Comment:
Please explain the reason for this in a code comment. (Is this the weird compatibility thing we have to do for drag-and-drop?)